src/components/cafes/[id]/MapBtn/index.tsx

```diff
@@ -0,0 +1,15 @@
+'use client';
+
+import { ReactNode } from 'react';
+import { mapBtn } from './MapBtn.css';
+
+export function MapBtn() {
```

Comment:
이름을 줄여쓰지 않고 Button으로 하면 어떨까요?  이미 머지된 내역중에 Button을 포함하는 네이밍이 있기도 하고, 대부분의 경우 풀어쓰는 경우가 많은 것 같아서 여기서만 줄이는 게 일관성이 떨어진 것 같아서요!
```suggestion
export function MapButton() {
```